<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>msControl</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Chan     Port</string>
   </property>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSlider" name="chanSlider">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="portSlider">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>141</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Prog     Vol     Mod     Pan</string>
   </property>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSlider" name="progSlider">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volSlider">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="modSlider">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="panSlider">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>81</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Pitch    Vel</string>
   </property>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QSlider" name="pitchSlider">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="velSlider">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>60</y>
      <width>24</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_6</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chanSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chanSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>portSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>portSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>progSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>modSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>panSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pitchSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
